saq/queue.py

```diff
@@ -18,14 +32,28 @@ class Queue:
     name: name of the queue
     dump: lambda that takes a dictionary and outputs bytes (default json.dumps)
     load: lambda that takes bytes and outputs a python dictionary (default json.loads)
+
+    The following parameters help prevent the Queue from consuming too many Redis connections.
+    enqueue_concurrency: max concurrent calls to enqueue jobs
+    get_job_concurrency: max concurrent calls to get jobs
+    abort_concurrency: max concurrent calls to abort jobs
     """
 
     @classmethod
-    def from_url(cls, url, name="default"):
+    def from_url(cls, url, **kwargs):
         """Create a queue with a redis url a name."""
-        return cls(aioredis.from_url(url), name)
-
-    def __init__(self, redis, name="default", dump=None, load=None):
+        return cls(aioredis.from_url(url), **kwargs)
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        redis,
+        name="default",
+        dump=None,
+        load=None,
+        enqueue_concurrency=100,
```

Comment:
i think this is overkill, can we just have one semaphore? for this? maybe default 20 

not sure of a good name

Reply:
Happy to switch to one, as long as we don't have any issues with reentrance. I don't think we have any such cases yet, but we'll have to keep an eye out.

As for an alternative name.... how about `max_concurrent_ops`?

Reply:
perfect